pages.de/windows/cipher.md

```diff
@@ -0,0 +1,16 @@
+# cipher
+
+> Verschlüsselt oder entschlüsselt Dateien auf NTFS-Laufwerken.
+> Weitere Informationen: <https://learn.microsoft.com/de-de/windows-server/administration/windows-commands/cipher>.
```

Comment:
We use language-agnostic links to keep all pages in sync. You will be forwarded anyways, if you have the correct locale set.
```suggestion
> Weitere Informationen: <https://learn.microsoft.com/windows-server/administration/windows-commands/cipher>.
```

Reply:
I will do a single commit with all the changes, thanks for your suggestions.